dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/ServiceConfigTests.java

```diff
@@ -175,8 +176,7 @@ public void testUpdateGrpcParam() throws Exception {
         assertTrue(greeting.getMessage().contains("Kevin"));
         //Make sure the Interceptor was not called and did not logged a message
         String interceptorHasRun = grpcServer.verifyStringNotInLogUsingMark("com.ibm.ws.grpc.fat.helloworld.service.HelloWorldServerInterceptor has been invoked!",
-                                                                            STARTUP_TIMEOUT);
-
+                                                                            SHORT_TIMEOUT);
         if (interceptorHasRun != null) {
             Assert.fail(c + ": server.xml with <grpc> element plus interceptor ran when it should not have in " + STARTUP_TIMEOUT + "ms");
```

Comment:
`STARTUP_TIMEOUT` should be corrected to `SHORT_TIMEOUT`

Reply:
Done